src/PlusImageProcessing/vtkPlusTransverseProcessEnhancer.cxx

```diff
@@ -32,58 +32,68 @@ vtkStandardNewMacro(vtkPlusTransverseProcessEnhancer);
-    UnprocessedLinesImage(vtkSmartPointer<vtkImageData>::New()),
-    ShadowImage(vtkSmartPointer<vtkImageData>::New()),
-    //ShadowFrame(),
-    //ShadowImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New()),
-    ShadowFrameList(vtkPlusTrackedFrameList::New()),
-    IntermediateImage(vtkSmartPointer<vtkImageData>::New()),
-    //IntermediateFrame(),
-    //IntermediateImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New()),
-    IntermediateFrameList(vtkPlusTrackedFrameList::New()),
-    ProcessedLinesImage(vtkSmartPointer<vtkImageData>::New()),
-    //ProcessedLinesFrame(),
-    //ProcessedLinesImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New())
-    ProcessedLinesFrameList(vtkPlusTrackedFrameList::New())
+  ConvertToLinesImage(false),
+  NumberOfScanLines(0),
+  NumberOfSamplesPerScanLine(0),
+
+  RadiusStartMm(0),
+  RadiusStopMm(0),
+  ThetaStartDeg(0),
+  ThetaStopDeg(0),
+
+  Thresholder(vtkSmartPointer<vtkImageThreshold>::New()),
+  GaussianEnabled(false),
+  ThresholdInValue(0.0),
+  ThresholdOutValue(255.0),
+  LowerThreshold(0.0),
+  UpperThreshold(0.0),
+
+  GaussianSmooth(vtkSmartPointer<vtkImageGaussianSmooth>::New()),
+  EdgeDetector(vtkSmartPointer<vtkImageSobel2D>::New()),
+  ImageBinarizer(vtkSmartPointer<vtkImageThreshold>::New()),
+  BinaryImageForMorphology(vtkSmartPointer<vtkImageData>::New()),
+  IslandRemover(vtkSmartPointer<vtkImageIslandRemoval2D>::New()),
+  ImageEroder(vtkSmartPointer<vtkImageDilateErode3D>::New()),
+
+  ReturnToFanImage(false),
+  CurrentFrameMean(0.0),
+  CurrentFrameStDev(0.0),
+  CurrentFrameMax(0.0),
+  CurrentFrameMin(255.0),
+
+  ThresholdingEnabled(false),
+
+  EdgeDetectorEnabled(false),
+  ConversionImage(vtkSmartPointer<vtkImageData>::New()),
+  IslandRemovalEnabled(false),
+  IslandAreaThreshold(-1),
+  ErosionEnabled(false),
+  DilationEnabled(false),
+  ReconvertBinaryToGreyscale(false),
+
+  LinesImage(vtkSmartPointer<vtkImageData>::New()),
+  //LinesFrame(),
+  //LinesImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New()),
+  LinesFrameList(vtkPlusTrackedFrameList::New()),
+  UnprocessedLinesImage(vtkSmartPointer<vtkImageData>::New()),
+  ShadowImage(vtkSmartPointer<vtkImageData>::New()),
+  //ShadowFrame(),
+  //ShadowImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New()),
+  ShadowFrameList(vtkPlusTrackedFrameList::New()),
+  IntermediateImage(vtkSmartPointer<vtkImageData>::New()),
+  //IntermediateFrame(),
+  //IntermediateImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New()),
+  IntermediateFrameList(vtkPlusTrackedFrameList::New()),
+  ProcessedLinesImage(vtkSmartPointer<vtkImageData>::New()),
+  //ProcessedLinesFrame(),
+  //ProcessedLinesImageList(vtkSmartPointer<vtkPlusTrackedFrameList>::New())
+  ProcessedLinesFrameList(vtkPlusTrackedFrameList::New())
 {
+  this->SetOutputImageSizePixel(0, 0);
+  this->SetTransducerCenterPixel(0, 0);
+  this->SetOutputImageSpacingMmPerPixel(0, 0);
+
+
   this->SetDilationKernelSize(0, 0);

```

Comment:
Include in the name if the kernel size in mm or in pixel and readius or diameter (e.g., SetDilationKernelDiameterMm or SetDilationKernelRadiusPixel)